### PR TITLE
Ww 80

### PR DESCRIPTION
**Pull request by HoXuanHieu**
this is a new branch for WW-80, I will delete the old branch

### CODE REVIEW

File: src/main/java/com/example/codeE/CodeEApplication.java [hunk 1]
```diff
@@ -6,7 +6,7 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
-@EnableSwagger2
+//@EnableSwagger2
```
Comment (th4tkh13m) [inline src/main/java/com/example/codeE/CodeEApplication.java:9]: LOL

Comment (HoXuanHieu) [reply src/main/java/com/example/codeE/CodeEApplication.java:9]: swagger dependency is having some errors now, WW-116 will resolve that.

**Comment by HoXuanHieu** [comment]
@Noro911 @th4tkh13m @th4tkh13m need to review again and merge

File: src/main/java/com/example/codeE/helper/ExcelHelper.java [hunk 2]
```diff
@@ -1,26 +1,59 @@
...
-    public static boolean isValidExcelFile(MultipartFile file){
+        public static boolean isValidExcelFile(MultipartFile file){
         return Objects.equals(file.getContentType(), "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet" );
     }
+     public static <T> void writeExcel(XSSFWorkbook workbook, List<T> dataList, String fileName, String sheetName){
+        try {
+            Sheet sheet = workbook.createSheet(sheetName);        
+            Row headerRow = sheet.createRow(0);
+            int cellIndex = 0;
+            for (java.lang.reflect.Field field : dataList.get(0).getClass().getDeclaredFields()) {
```
Comment (TungPhan2808) [inline src/main/java/com/example/codeE/helper/ExcelHelper.java]: remove "java.lang.reflect."

File: src/main/java/com/example/codeE/helper/ExcelHelper.java [hunk 3]
```diff
@@ -1,26 +1,59 @@
...
+                Cell cell = headerRow.createCell(cellIndex++);
+                cell.setCellValue(field.getName());
+            }
 
+            int rowIndex = 1;
+            for (Object data : dataList) {
+                Row row = sheet.createRow(rowIndex++);
+                cellIndex = 0;
+                for (java.lang.reflect.Field field : data.getClass().getDeclaredFields()) {
```
Comment (TungPhan2808) [inline src/main/java/com/example/codeE/helper/ExcelHelper.java:41]: remove "java.lang.reflect."

File: src/main/java/com/example/codeE/helper/ExcelHelper.java [hunk 4]
```diff
@@ -1,26 +1,59 @@
...
+                    field.setAccessible(true);
+                    Cell cell = row.createCell(cellIndex++);
+                    cell.setCellValue(String.valueOf(field.get(data)));
```
Comment (TungPhan2808) [inline src/main/java/com/example/codeE/helper/ExcelHelper.java:44]: for example, if we export users, do we also export the hash pass in the excel file too? 
is that ok to show hash pass or any sensitive information?

File: src/main/java/com/example/codeE/helper/ExcelHelper.java [hunk 5]
```diff
@@ -1,26 +1,59 @@
...
+import org.springframework.web.multipart.MultipartFile;
+
+import com.example.codeE.constant.Constant;
+
 public class ExcelHelper {
-    public static boolean isValidExcelFile(MultipartFile file){
+        public static boolean isValidExcelFile(MultipartFile file){
         return Objects.equals(file.getContentType(), "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet" );
     }
+     public static <T> void writeExcel(XSSFWorkbook workbook, List<T> dataList, String fileName, String sheetName){
```
Comment (TungPhan2808) [inline src/main/java/com/example/codeE/helper/ExcelHelper.java:27]: why do you use void? I believe the purpose of the export function is to export Excel files.

File: src/main/java/com/example/codeE/helper/ExcelHelper.java (hunk 4 above)
Comment (th4tkh13m) [reply src/main/java/com/example/codeE/helper/ExcelHelper.java:44]: Why do we export password? It is meaningless.

File: src/main/java/com/example/codeE/helper/ExcelHelper.java (hunk 3 above)
Comment (HoXuanHieu) [reply src/main/java/com/example/codeE/helper/ExcelHelper.java:41]: done

**Review by Noro911 — APPROVED** [review]

**Pull request merged**